app/src/main/java/io/intrepid/contest/screens/conteststatus/ContestStatusContract.java

```diff
@@ -0,0 +1,14 @@
+package io.intrepid.contest.screens.conteststatus;
+
+import io.intrepid.contest.base.BaseContract;
+
+class ContestStatusContract {
+    interface View extends BaseContract.View {
+        void showWaitingSubmissionsFragment(int numSubmissionsMissing);
+
+        void showApiErrorMessage();
+    }
+
+    interface Presenter extends BaseContract.Presenter<View> {
```

Comment:
For now the view does not call any methods from the presenter, everything is done from presenter's `onViewCreated` and keeps refreshing from there.
I have to keep the `Presenter` interface in the Contract because with `BaseMvpActivity` my `View` still gets assigned a presenter of this interface type. Also it's much easier later just to add method signatures there if ever needed.